extensions/amp-carousel/amp-carousel.md

```diff
@@ -72,35 +73,36 @@ Usage example:
 ```
 
 **type**
-- `carousel` (default) - All slides are shown and are scrollable horizontally.
-  Be aware that `type=carousel` does not currently support `layout=responsive`.
-- `slides` - Shows a single slide at a time. Supports `layout=responsive`.
+
+- `carousel` (default): All slides are shown and are scrollable horizontally.
+  The `carousel` type only supports `layout="fixed"` and `layout="fixed-height"`.
+- `slides`: Shows a single slide at a time. 
```

Comment:
no NODISPLAY means this component will not be visible at any time. It is not supported per the code.